### ISSUE TITLE
about network

### ISSUE THREAD

**Issue by suxi-du** [cao-cong/RViDeNet#3]
hello, I'm interested in this work, too. The questions are as follows: 1) What does raw domain processing mean in ablation study? If we don't use raw domain processing, what deos the network look like? Is it  EDVR?  2) Have you tried other ISP model? I used U-net to demosaick, but the result is not so good, even worse than 8 residual blocks. 3)  how do you retrain  TO-flow? using 3  or 7 sRGB images as input?  I hope to hear from you soon~

**Comment by cao-cong** [comment]
1. Raw domain processing means training and testing are directly performed in raw domain rather than sRGB domain. If we remove raw domain processing, packing, predenoising, and nonlocal attention, our model will look like EVDR, but there is CBAM module in reconstruction part and the loss functions contain consistency loss in sRGB domain.
2. No, we did not try other ISP models.  This work focuses on the denoising process not the ISP process. Maybe you can refer to other paper like 'AIM 2019 Challenge on RAW to RGB Mapping: Methods and Results (2019-ICCV)' for the ISP process.
3. We used 7 sRGB images as input. The sRGB images are generated from the raw video dataset (including the synthesized raw videos (for pretraining) and captured raw videos (for finetuning) ) by our pretrained ISP module. We used code from https://github.com/Coldog2333/pytoflow with its default settings.

**Comment by suxi-du** [comment]
ok, thanks.
